src/ontology/mondo-edit.obo

```diff
@@ -82711,16 +82711,16 @@ intersection_of: disease_has_location UBERON:0000310 ! breast
 
 [Term]
 id: MONDO:0004990
-name: breast tumor luminal A or B
-def: "A biologic subset of breast carcinoma defined by low to moderate expression of genes characteristic of luminal epithelial cells including estrogen receptor (ER), and high expression of GGH, LAPTM4B, and CCNE1. This subtype of breast cancer is associated with a good prognosis, although not as favorable as the luminal A subtype." [NCIT:P378]
+name: obsolete breast tumor luminal A or B
 synonym: "breast tumor luminal" EXACT [DOID:0060548]
 synonym: "luminal breast cancer" EXACT [DOID:0060548]
 synonym: "luminal breast carcinoma" RELATED [DOID:0060548]
 xref: DOID:0060548 {source="MONDO:equivalentTo"}
```

Comment:
surprised travis isn't complaining - should be obsoleteEquivalent

Reply:
fixed